test/jdk/com/sun/jndi/dns/Test6991580.java

```diff
@@ -62,8 +62,10 @@ private static void init() throws Exception {
          "Modify the /etc/resolv.conf file if needed. "
             + "Don't forget to save the original content of the file.",
          "3. Type \"cd " + System.getProperty("test.classes") + "\".",
-         "4. Type \"" + System.getProperty("java.home") +
-                "/bin/java IPv6NameserverPlatformParsingTest\".",
+         "4. Type \"" + System.getProperty("java.home") + "/bin/java"
+                 + " --add-modules java.desktop,jdk.naming.dns"
```

Comment:
I don't think you'll need the --add-modules in the instructions. The java.desktop exports as an API so it will be resolved by default, same for the java.naming and that will ensure that the JNDI service provider modules are resolved.

Reply:
Thank you for the feedback. I will remove the --add-modules.